src/nisarqa/processing/__init__.py

```diff
@@ -0,0 +1,14 @@
+from .utils_for_plotting import *
+from .utils_for_processing import *
```

Comment:
Consider these alternative names for brevity

```suggestion
from .plotting_utils import *
from .processing_utils import *
```